linux.md

```diff
@@ -0,0 +1,39 @@
+# .NET for Linux Distributions
+
+.NET can be installed on Linux in multiple ways, via [packages](https://learn.microsoft.com/dotnet/core/install/linux), [scripts](https://github.com/dotnet/install-scripts), or [tarballs](https://dotnet.microsoft.com/download/dotnet). It can be installed globally, locally/privately, or with an app.
+
+.NET is supported on Linux per [.NET Support and Compatibility for Linux Distributions](linux-support.md).
+
+## Official archives
```

Comment:
Would it be appropriate to include a small caution note in this section for any customers migrating from packages.microsoft.com usage?

Reply:
Yes. Good call.

We could link to this, or is it too specific? https://github.com/dotnet/core/issues/7699

Reply:
There's also https://github.com/dotnet/docs/blob/main/docs/core/install/linux-package-mixup.md that provides similar info